Makefile

```diff
@@ -49,6 +50,9 @@ out/delta.tar.gz: bin/init bin/vsockexec bin/cmd/gcs bin/cmd/gcstools bin/cmd/ho
 	cp bin/cmd/gcs rootfs/bin/
 	cp bin/cmd/gcstools rootfs/bin/
 	cp bin/cmd/hooks/wait-paths rootfs/bin/
+	if [ $(INCLUDE_UTILITIES) -eq 1 ]; then \
```

Comment:
Maybe a separate target like `out/delta-dev.tar.gz` would work better? Then we could keep `snp-report` off the deps list for the main target.

Reply:
good idea!

Reply:
I've added a different `DEV_BUILD` parameter and a new `out/delta-dev.tar.gz` target. When `DEV_BUILD=1` we build `out/delta-dev.tar.gz` (instead of `out/delta.tar.gz`) and use it when building initrd.img and rootfs.tar.gz.